scripts/asm-installer/asm_vm

```diff
@@ -1384,11 +1403,22 @@ gen_gce_instance_labels() {
 
 # Generate a default instance template.
 gen_default_instance_template() {
+  local source_image="projects/debian-cloud/global/images/family/debian-10"
+  if [[ ! -z "${IMAGE_FAMILY}" ]]; then
+    if [[ "${IMAGE_FAMILY}" == "debian-10" ]] || [[ "${IMAGE_FAMILY}" == "debian-9" ]]; then
+      source_image="projects/debian-cloud/global/images/family/${IMAGE_FAMILY}"
+    elif [[ "${IMAGE_FAMILY}" == "centos-7" ]] || [[ "${IMAGE_FAMILY}" == "centos-8" ]]; then
+      source_image="projects/centos-cloud/global/images/family/${IMAGE_FAMILY}"
+    else
+      fatal "${IMAGE_FAMILY} is not supported."
```

Comment:
This validation should happening at the beginning of the script execution.

Reply:
ok will add that